bricks/nav/src/data-providers/shared/smartDisplayForMenuTitle.ts

```
@@ -0,0 +1,108 @@
+import type { MicroApp } from "@next-core/types";
+import { i18n } from "@next-core/i18n";
+import { hasOwnProperty } from "@next-core/utils/general";
```

Biome (1.9.4)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)